test/libsolidity/syntaxTests/dataLocations/state_variable_storage_transient.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    uint storage transient x;
```

Comment:
Would be good to cover this too (should fail):
```solidity
    uint storage transient;
```

Reply:
And these (all should work):
```solidity
    function () transient f;
    function () transient internal fti;
    function () internal transient fit;
    function () internal transient internal fiti;
    function () internal internal transient fiit;
```

Reply:
Done.